src/routes/subpage.jsx

```diff
@@ -0,0 +1,26 @@
+import TitleTextAndImg from "../TitleTextImg.jsx";
+import {Link} from "react-router-dom";
+import HomeButton from "../HomeButton.jsx";
+
+export function SubPage() {
+    return (
+        <>
+            <HomeButton/>
+            <TitleTextAndImg
+                title="This is a subpage"
+                text="yes, a subpage this is"
+                img_source="https://www.eology.net/wp-content/uploads/2021/10/Graphic-Subpage-edited.png"/>
+        </>
+    )
+}
+
+export function RootSubpage() {
```

Comment:
Generally, we would stick to one component per file (with some exceptions). This is fine for just playing about with react/react router however